DependencyInjection/FOSRestExtension.php

```diff
@@ -163,9 +164,34 @@ private function loadFormatListener(array $config, XmlFileLoader $loader, Contai
                 $service->clearTag('kernel.event_listener');
             }
 
-            foreach ($config['format_listener']['rules'] as $key => $rule) {
+            foreach ($config['format_listener']['rules'] as &$rule) {
                 if (!isset($rule['exception_fallback_format'])) {
-                    $config['format_listener']['rules'][$key]['exception_fallback_format'] = $rule['fallback_format'];
+                    $rule['exception_fallback_format'] = $rule['fallback_format'];
+                }
+                $this->addFormatListenerRule($rule, $config, $container);
+            }
+
+            $bundles = $container->getParameter('kernel.bundles');
+            if (isset($bundles['WebProfilerBundle'])) {
+                $profilerConfig = $this->processConfiguration(new ProfilerConfiguration(), $configs);
```

Comment:
I am not sure how this is supposed to work. At least, there should be a test for it.